### PR TITLE
Adding overloaded method to allow a specific credit card format to be returned

### PR DESCRIPTION
**Pull request by shortstacked**
This method allows specific credit card types to be returned. This is useful for testing purposes when someone needs to ensure they test only a certain subset of credit cards, e.g Visa and Mastercard, but not AMEX.

Added unit test.

### CODE REVIEW

File: src/main/java/com/github/javafaker/Finance.java [hunk 1]
```diff
@@ -18,10 +18,10 @@ protected Finance(Faker faker) {
 
     private static final Map<String, String> countryCodeToBasicBankAccountNumberPattern =
             createCountryCodeToBasicBankAccountNumberPatternMap();
-//
-    public String creditCard() {
-        CreditCardType type = randomCreditCardType();
-        final String key = String.format("credit_card.%s", type.toString().toLowerCase());
+
+    public String creditCard(CreditCardType creditCardType)
+    {
```
Comment (codingricky) [inline src/main/java/com/github/javafaker/Finance.java]: can you keep the brace formatting the same please?

Comment (shortstacked) [reply src/main/java/com/github/javafaker/Finance.java]: Yup, fixed that. Thanks.

File: src/test/java/com/github/javafaker/FinanceTest.java [hunk 2]
```diff
@@ -32,4 +36,13 @@ public void iban() {
     public void ibanWithCountryCode() {
         assertThat(faker.finance().iban("DE"), matchesRegularExpression("DE\\d{20}"));
     }
+
+    @Test
+    public void creditCardWithType() {
+        for(CreditCardType type : CreditCardType.values())
+        {
```
Comment (codingricky) [inline src/test/java/com/github/javafaker/FinanceTest.java]: One more? Sorry!

Comment (shortstacked) [reply src/test/java/com/github/javafaker/FinanceTest.java]: Ah yes, found it! Thanks.

**Pull request merged**